README.md

```diff
@@ -1 +1,61 @@
 # Javascript Group Capstone
+Description the project.
+
+Built With
+* Major languages
+* Frameworks
+* Technologies used
+
+Live Demo (if available)
+Live Demo Link
```

Comment:
- [ ] [REQUIRED] Please, as requested in project requirements, add a link to the presentation video.

1. The student recorded a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video contains a clear objective. 
4. description of the project in professional English. The topic and purpose of the project are made clear.
5. All the members in the video should appear in the video and talk about the project.
6. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.